functions/src/modules/SQR.functions.ts

```diff
@@ -512,4 +512,16 @@ export const exportSubmissionsToSpreadsheet = functions.database
         'Email Address': withDefault(author.emailAddress),
       });
     }
-  );
+  )
+
+export const processSQRDoneFromGmail = async (processObject: any) => {
```

Comment:
We are already in the SQR module here. No need specifying `SQR` inside the function name. You can namespace the function while importing in another module, as I suggested later in the code.
```suggestion
export const processMarkingSubmissionAsDone = async (processObject: any) => {
```

Reply:
Yes, it's changed.